larva/src/main/java/org/frankframework/larva/output/PlainTextScenarioOutputRenderer.java

```diff
@@ -61,6 +62,9 @@ public void executionOverview(TestRunStatus testRunStatus, long executionTime) {
 	@Override
 	public void startScenario(TestRunStatus testRunStatus, Scenario scenario) {
 		// No-op
+		System.err.println("log.dir=" + System.getProperty("log.dir"));
+		System.err.println("log.dir=" + AppConstants.getInstance().getProperty("log.dir"));
```

Comment:
is het handig deze twee de zelfde 'key' te geven?

Reply:
Oh, dat is wat debug-output die ik had toegevoegd omdat ik wou weten waarom iets mis ging. Was niet de bedoeling om dat te committen 🫣